### PR TITLE
[master] Bump OVN to 20.09.0-7

### PR DESCRIPTION
**Pull request by dcbw**
We have good signal for this build in 4.6 via https://github.com/openshift/ovn-kubernetes/pull/324 so we're pretty sure this OVN is solid.

@trozet

### CODE REVIEW

**Comment by dcbw** [comment]
(need to wait for brew build to propagate to rhaos tags)

**Comment by trozet** [comment]
Error: Unable to find a match: ovn2.13 == 20.09.0-7.el8fdn ovn2.13-central == 20.09.0-7.el8fdn ovn2.13-host == 20.09.0-7.el8fdn ovn2.13-vtep == 20.09.0-7.el8fdn

**Comment by trozet** [comment]
oh just saw your comment to wait

**Comment by dcbw** [comment]
Still stuck on some issues getting the 4.7 repos synced into CI

File: Dockerfile [hunk 1]
```diff
@@ -32,18 +32,19 @@ RUN yum install -y  \
...
 	tcpdump iputils \
 	" && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS && \
-	#======== 4.6.0-0.nightly-2020-09-25-070943 Version ========
-	# ovn2.13-20.06.2-11.el8fdp.x86_64
-	# openvswitch2.13-2.13.0-57.el8fdp
-	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "openvswitch2.13 == 2.13.0-57.el8fdp" "openvswitch2.13-devel == 2.13.0-57.el8fdp" "python3-openvswitch2.13 == 2.13.0-57.el8fdp" && \
-	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "ovn2.13 == 20.06.2-11.el8fdp" "ovn2.13-central == 20.06.2-11.el8fdp" "ovn2.13-host == 20.06.2-11.el8fdp" "ovn2.13-vtep == 20.06.2-11.el8fdp" && \
+	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False "openvswitch2.13 == $ovsver" "openvswitch2.13-devel == $ovsver" "python3-openvswitch2.13 == $ovsver" && \
+	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False ovn2.13 && \
```
Comment (trozet) [inline Dockerfile:46]: @dcbw this shouldn't be here right?

Comment (dcbw) [reply Dockerfile:46]: Yeah, it was testing stuff. We figured out what the issue was in openshift/release though (4.7 was still pointed at 4.6 RPMs).

**Comment by dcbw** [comment]
@trozet PTAL thanks

**Review by trozet — APPROVED** [review]
/lgtm

**Pull request merged**